src/views/ABTesting.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div></div>
+</template>
+
+<script>
+import axios from "axios";
+
+export default {
+    name: "ABTesting",
+    data() {
+        return {
+            assignment: null,
+            userConfig: {},
+            plioId: null
+        };
+    },
+
+    created() {
+        if(!localStorage.phone) {
+            this.$router.push({
+                path: '/login/' + this.$route.params.id + '/experiment'
+            })
+        } else {
+            // get variant and redirect user
+            this.getAssignment();
+        }
+    },
+    methods: {
+        getAssignment() {
+            var url = process.env.VUE_APP_BACKEND + 
+                process.env.VUE_APP_BACKEND_EXPERIMENT_ASSIGNMENT +
+                "?experimentId=" + this.$route.params.id +
+                "&userId=" + localStorage.phone;
+            axios
+                .get(url)
+                .then((res) => {
+                    // separately seting plio ID although it will be the
+                    // same as assignment for now as we might conduct interface
+                    // level changes where assignment won't be the same as plio ID
+                    this.assignment = res.data.assignment;
+                    this.userConfig = res.data.config;
```

Comment:
Rename this variable to `userExperimentsConfig` because (if I'm understanding correctly) this is only config for experiments no?

Reply:
For now, this is retrieving the whole config (it will become clearer in the backend ka PR). The point was that the user should only need to fetch the config once. IF they are coming through the experiment link, then we fetch the config here and store it in Vuex. Now, when Deepansh's code wants to check for the tutorial progress, he will first check in 
Vuex if there is a config in Vuex. If it is present, then it simply uses it. Does that make sense? I thought we'll make the segregation amongst configs when we feel the need to.

Reply:
Haan understood. Feels a little strange... Can you just add a comment in the `Login` component and router that individual components are responsible for getting the user's config if they need it.

Reply:
I gave Deepansh a suggestion on the phone. Make a separate component to handle user config stuff. Then any other component can say `UserConfigComponent.fetchConfig("experiments")` and the Config component can handle fetching and all that depending on the state.

It's better to do it now than refactor later. But if some deadline

Reply:
@dalmia what do you think?

Reply:
UserConfig please thoda alag se hi karte hai please